src/sdk/sql_cluster_router.h

```diff
@@ -152,6 +154,10 @@ class SQLClusterRouter : public SQLRouter {
     std::shared_ptr<hybridse::sdk::ResultSet> ExecuteSQLRequest(const std::string& db, const std::string& sql,
                                                                 std::shared_ptr<SQLRequestRow> row,
                                                                 hybridse::sdk::Status* status) override;
+
+    std::shared_ptr<hybridse::sdk::ResultSet> HandleSQL(const std::string& sql,
```

Comment:
Is it better to be `ExecuteSql`? We may rename `ExecuteSQL` as `QuerySql` later.